static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/main/variant.js

```diff
@@ -31,6 +32,7 @@ const canShow: (name?: string) => boolean = (name = '') => {
         !isLoggedIn() &&
         !isInvalidArticleType() &&
         !isInvalidSection() &&
+        !isInvalidTag() &&
```

Comment:
you'll want to add this to the main/control variant too, since although we dont show a gate on the control variant, we still run the display rules so we can compare both variants directly